### PR TITLE
Add follow/unfollow commands

### PR DESCRIPTION
**Pull request by prologic**

### CODE REVIEW

File: config.go [hunk 1]
```diff
@@ -24,6 +26,20 @@ type Config struct {
 	DiscloseIdentity bool
 	Hooks            Hooks
 	nicks            map[string]string // normalizeURL(url) -> nick
+	path             string            // location of loaded config
+}
+
+func (conf *Config) Write() error {
+	if conf.path == "" {
+		return errors.New("errorr: no config file path found")
```
Comment (quite) [inline config.go]: spelling

File: config.go [hunk 2]
```diff
@@ -24,6 +26,20 @@ type Config struct {
 	DiscloseIdentity bool
 	Hooks            Hooks
 	nicks            map[string]string // normalizeURL(url) -> nick
+	path             string            // location of loaded config
```
Comment (quite) [inline config.go:29]: This is a good idea. Wonder if we could get rid of the configpath global...

**Review by quite — CHANGES_REQUESTED** [review]

**Review by quite — COMMENTED** [review]
Please also add a redirect from main's `case "help"`.

**Comment by quite** [comment]
Please also add a redirect from main's `case "help"`.

File: config.go (hunk 2 above)
Comment (prologic) [reply config.go:29]: Possibly IF I can i'll do so in a "cleanup" PR.

**Review by quite — APPROVED** [review]

**Pull request merged**